### DISCUSSION TITLE
Purchased a license 24+ months ago, where do I find it?

### DISCUSSION THREAD

**Discussion by MasterZii** [aclap-dev/video-downloadhelper#471] [License & Purchase requests]
### What's your issue?

So I am logged into my google account, let's call it GoogleAccountA. This is the same account I purchased the license on (Chrome).

I have the extension installed on my new PC build, and tried to download a few videos. Got a message saying I need a license if I want to do it every 120 minutes or something. 

Not sure how to re-enable my license. I searched my email inbox for GoogleAccountA and my paypal far and wide and do not see any serials. It has been a long time so I am not even sure which year to search under. 

How can I locate my license to reactivate premium? 

### The 10 first digits (only 10) of your license number or email address

grandmaste

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
hi
please read this: https://github.com/aclap-dev/video-downloadhelper/wiki/Old-Chrome-licenses-(key-missing)
